test/test_samples_osw/base/results_output.csv

```diff
@@ -1,237 +1,237 @@
-project_national-2934.osw,20.0,338.6,11.0,0.0,66.6,35.1,65.4,634.9,0.0,0.0,40.3,0.0,227.1,124.2,0.0,0.0,39.5,0.0,0.0,0.0,0.0,491.2,1980.4,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,434.3,0.0,0.0,0.0,610.0,1220.0,1220.0,0.0,0.0,6.0,1.0,5696.6,7690.4,0.0,true,30.2,23.2,0.0,0.0,147.5,0.0,0.0,0.0,144.1,0.0,0.0,0.0,0.0,682.0,19.4,26.2,40.0,4160.9,48.7,0.0,345.0,0.0,0.0,1649.4,1734.2,0.0,197.9,0.0
-project_national-2945.osw,20.0,338.6,5.5,0.0,33.4,18.7,42.9,4740.0,0.0,0.0,40.3,0.0,1285.2,134.2,0.0,0.0,26.8,0.0,691.5,682.7,0.0,491.2,2045.2,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,480.4,0.0,0.0,0.0,610.0,1220.0,1220.0,0.0,0.0,0.0,41.0,13521.7,12245.2,0.0,true,15.2,15.2,0.0,0.0,159.2,0.0,0.0,0.0,93.6,0.0,0.0,0.0,0.0,682.0,46.1,41.8,30.0,10729.0,64.9,0.0,283.1,0.0,0.0,1649.4,1734.2,0.0,148.5,0.0
-project_national-2958.osw,20.0,469.0,11.0,111.1,43.7,23.0,42.9,3721.3,44.6,0.0,184.5,480.4,0.0,1081.3,0.0,0.0,85.3,0.0,0.0,0.0,0.0,1938.9,1658.3,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,480.4,0.0,0.0,0.0,845.0,1690.0,1690.0,0.0,0.0,15.0,0.0,13562.7,30473.7,0.0,true,19.8,15.2,0.0,0.0,1459.9,0.0,0.0,0.0,117.4,0.0,0.0,0.0,0.0,944.7,46.3,104.0,40.0,9917.6,195.0,0.0,1612.3,0.0,0.0,1941.3,2058.7,970.7,233.0,0.0
-project_national-2969.osw,20.0,0.0,5.5,147.0,0.0,0.0,45.5,953.2,0.0,0.0,142.7,0.0,342.9,129.8,0.0,0.0,95.1,0.0,0.0,0.0,0.0,1220.4,1231.3,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,434.3,0.0,45.0,0.0,0.0,853.0,853.0,0.0,0.0,43.0,180.0,2957.3,3283.7,0.0,true,0.0,16.1,0.0,0.0,175.3,0.0,0.0,0.0,95.3,0.0,0.0,0.0,0.0,0.0,10.1,11.2,30.0,4803.6,45.1,0.0,286.8,0.0,0.0,174.2,174.2,0.0,52.3,0.0
-project_national-2971.osw,20.0,390.4,11.0,73.9,1132.7,47.2,87.9,899.4,0.0,0.0,26.8,0.0,330.8,161.8,0.0,0.0,41.9,0.0,0.0,0.0,0.0,375.2,2259.5,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,434.3,0.0,0.0,383.5,1220.0,1220.0,1220.0,0.0,0.0,1.0,0.0,8773.4,9434.7,0.0,true,0.0,31.3,0.0,0.0,222.0,0.0,0.0,0.0,174.0,0.0,0.0,0.0,0.0,1364.0,29.9,32.2,40.0,6277.0,64.1,0.0,427.4,0.0,0.0,1166.3,1335.8,0.0,175.0,0.0
-project_national-2988.osw,20.0,390.4,11.0,30.5,66.6,35.1,409.5,2505.8,68.0,0.0,161.0,0.0,685.7,340.2,0.0,0.0,46.8,0.0,0.0,0.0,0.0,1535.4,1790.8,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,434.3,0.0,0.0,0.0,1220.0,1220.0,1220.0,0.0,0.0,0.0,0.0,14473.0,17486.3,0.0,true,30.2,0.0,0.0,0.0,459.3,0.0,0.0,0.0,173.7,0.0,0.0,0.0,0.0,1364.0,49.4,59.7,40.0,8131.7,94.1,0.0,663.2,0.0,0.0,1166.3,1335.8,0.0,140.0,0.0
+project_national-0009.osw,20.0,540.8,11.0,0.0,66.6,35.1,65.4,1090.0,90.9,0.0,30.8,0.0,497.2,52.7,0.0,16.7,19.6,0.0,0.0,0.0,0.0,473.7,2113.0,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,434.3,0.0,0.0,0.0,1690.0,1690.0,1978.0,0.0,0.0,0.0,1.0,10072.9,6582.7,0.0,true,30.2,23.2,0.0,0.0,62.5,0.0,0.0,0.0,148.4,0.0,0.0,0.0,0.0,2211.5,34.4,22.5,40.0,5007.9,43.5,0.0,264.3,0.0,0.0,1430.0,1894.7,0.0,111.4,0.0
```

Comment:
@shorowit These (big) changes are expected?

Reply:
They appear to have been introduced by https://github.com/NREL/resstock/pull/615, which modified TSVs and thus caused samples to change.